scripts/check-size.js

```diff
@@ -9,7 +9,7 @@ import * as fs from 'fs';
 // it's likely that we'll ask you to investigate ways to reduce the size.
 //
 // In either case, update the size here and push a new commit to your PR.
-const expectedLitCoreSize = 15023;
+const expectedLitCoreSize = 15135;
```

Comment:
Huh, bigger change than I'd have expected from this PR

Reply:
Yeah... there's a bit of change here. I'm going to change the approach slightly, though I don't think it'll be fewer bytes

Reply:
my new change made it worse: 15214 bytes, but I think aside from that it's a better implementation.

Reply:
Ok, kept it more static and got the size down to 15115